src/include/miopen/conv/problem_description.hpp

```diff
@@ -338,8 +338,10 @@ struct ProblemDescription
     }
     bool IsInt8() const
     {
-        return GetInDataType() == miopenInt8 && GetWeightsDataType() == miopenInt8 &&
-               (GetOutDataType() == miopenInt32 || GetOutDataType() == miopenFloat);
+        return (GetInDataType() == miopenInt8 && GetWeightsDataType() == miopenInt8 &&
+                (GetOutDataType() == miopenInt32 || GetOutDataType() == miopenFloat)) ||
+               (GetOutDataType() == miopenInt8 && GetWeightsDataType() == miopenInt8 &&
+                (GetInDataType() == miopenInt32 || GetInDataType() == miopenFloat));
```

Comment:
🐛 This enables FWD convolutions with INT32 inputs and INT8 outputs, but these are not supported by the library IIRC.

Reply:
This is for the BWD convolution. I think this if condition won't enable FWD convolutions with INT32 input, as INT8 weight is requested here, driver will help us prevent this things happen: FWD convolution weight and input are in different data type.

Reply:
Extending from this: I don't think we need `bwd` or `wrw` for int8 because IIRC no one will actually do backward pass in int8 type.

Reply:
Understand.
Let me disable the bwd support for int8.

Reply:
Hi @atamazov , @jerryyin , @JehandadKhan 
bwd for int8 has been disabled.